sematic/utils/__init__.py

```diff
@@ -0,0 +1,16 @@
+# Standard Library
```

Comment:
instead, can this be in `sematic/utils/cli_arg_conversion.py`? I imagine there will be other things we will want to convert from CLI args, and then it's not in a file that's as likely to be a dumping ground.

Reply:
Let's keep this as a private function of `user_settings.py` and have `get_bool_user_settings` to wrap around it as suggested above.